### DISCUSSION TITLE
treeTable删除子级后图标不同步

### DISCUSSION THREAD

**Discussion by hjingsuper** [layui/layui#1782] [讨论]
示例：[https://codepen.io/hjingsuper/pen/yLrEeNV](https://codepen.io/hjingsuper/pen/yLrEeNV)
![2024-04-11-16-35-34](https://github.com/layui/layui/assets/36070786/bc602bde-a0f3-4aff-aeb4-0bf5446fed82)

不知道是不是【删除节点时，内部缓存数据同步的 BUG】的原因，删除子节点后，父节点【折叠图标】、【节点图标】不会更新

ref:
#1752 
#1753

**Comment by Sight-wcg** [comment]
和 “删除节点时，内部缓存数据同步的 BUG” 无关，我在调试 #1753 时也有发现其他问题，但还未处理。确认是 BUG, 你可提一个新的 isuess 吗？后续版本会修复。

**Reply by hjingsuper** [reply]
已创建isuess: #1785
